docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -7,74 +7,221 @@
+status. 
+
+[float]
+[[df-ui-limitation]]
+=== {dataframes} UI will not work during a rolling upgrade from 7.2 to 7.3
+
+If your cluster contains mixed version nodes, for example during a rolling 
+upgrade from 7.2 to 7.3, and {dataframe-transforms} have been created in 7.2, 
+the {dataframe} UI will not work. Please wait until all nodes have been upgraded 
+to 7.3 before using the {dataframe} UI.
+
+
 [float]
 [[df-datatype-limitations]]
 === {dataframe-cap} data type limitation
 
 {dataframes-cap} do not (yet) support fields containing arrays – in the UI or 
-the API. If you try to create one, the UI will fail to show the source index table.
+the API. If you try to create one, the UI will fail to show the source index 
+table.
 
 [float]
 [[df-ccs-limitations]]
-=== {ccs-cap} limitation
+=== {ccs-cap} is not supported
 
-{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+{ccs-cap} is not supported in 7.3 for {dataframe-transforms}.
 
 [float]
 [[df-kibana-limitations]]
-=== {kib} only displays up to 100 {dataframe-transforms}
+=== Up to 1,000 {dataframe-transforms} are supported
 
-The {kib} *Machine Learning* > *Data Frames* page lists up to 100 
-{dataframe-transforms}. You can work-around this limitation by calling the 
-{ref}/get-data-frame-transform.html[GET {dataframe-transforms} API] 
-with the `size` parameter.
+A single cluster will support up to 1,000 data frame transforms.
+When using the 
+{ref}/get-data-frame-transform.html[GET {dataframe-transforms} API] a total 
+`count` of transforms is returned. Use the `size` and `from` parameters to 
+enumerate through the full list.
 
 [float]
-[[df-dateformat-limitations]]
-=== Date histogram limitation
+[[df-aggresponse-limitations]]
+=== Aggregation responses may be incompatible with destination index mappings
 
-If you use a {ref}/search-aggregations-bucket-datehistogram-aggregation.html[date 
-histogram] in the `group_by` object in the create or preview {dataframe-transform} 
-APIs, the defined interval and time format must have the same time fidelity. 
-Otherwise, it might cause issues in the {dataframe}.
+When a {dataframe} is first started, it will deduce the mappings required for the 
+destination index. This process is based on the field types of the source index 
+and the aggregations used, or in the case of fields derived from 
+{ref}/search-aggregations-metrics-scripted-metric-aggregation.html[`scripted_metrics`] 
+or {ref}/search-aggregations-pipeline-bucket-script-aggregation.html[`bucket_scripts`], 
+{ref}/dynamic-mapping.html[dynamic mappings] will be used. In some instances the 
+deduced mappings may be incompatible with the actual data. For example, with 
+possible numeric overflows or dynamically mapped fields containing both numbers 
+and strings. Please check {es} logs if you think this may have occurred. As a 
+workaround, you may define custom mappings prior to starting the 
+{dataframe-transform}. Do this by {ref}/indices-create-index.html[creating a 
```

Comment:
Do we really need to explain the use of deduced or dynamic mappings?  If not, can simplify to something like this: "For example, (link)create a custom destination index or (link)define an index template".